src/App.test.js

```diff
@@ -1,8 +1,18 @@
-import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { render, screen, waitFor, fireEvent, getByTestId } from '@testing-library/react';
```

Comment:
I think that you don't need to explicitly  import 'getByTestId'

Reply:
x2

Reply:
Yeah you are right, thanks